GoogleForCreators/web-stories-wp
# Embeds: Improve default WordPress oEmbed appearance

## Context



## Summary

Override the embed.php template shipped in core, with our own version. If poster image is set, use the Image render class otherwise fallback to built in embed. 

## Relevant Technical Choices



## To-do



## User-facing changes

### Embed with poster
https://user-images.githubusercontent.com/237508/113140064-cfa0de80-921f-11eb-8e5f-2b621c80a8ae.mov
### Embed without poster
<img width="1904" alt="Screenshot 2021-03-31 at 12 29 44" src="https://user-images.githubusercontent.com/237508/113140173-f3fcbb00-921f-11eb-8605-17d7022974ca.png">


## Testing Instructions



### QA






This PR can be tested by following these steps:

Gutenberg test
1. Create a story and add a poster. 
2. Publish story. 
3. View story and copy the link to the story to clipboard. 
4. Create a post
5. Add an embed block. 
6. Paste link to story and click embed. 
7. Check story renders in block editor. 
8. View post on front end. 

Classic editor test. 
1. Create a story and add a poster. 
2. Publish story. 
3. View story and copy the link to the story to clipboard. 
4. Activate classic editor plugin
4. Create a poster 
6. Paste link to story in content
7. Check story renders in editor
8. View post on front end. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #1687
